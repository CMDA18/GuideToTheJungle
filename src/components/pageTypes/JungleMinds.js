// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
import styled from 'styled-components'
import Navigation from '../Navigation'
import MainContentGrid from '../MainContentGrid'
import PageTitle from '../PageTitle'
import Intro from '../Intro'
import { jungleMinds, overallData } from '../../constants/content/index'
import { TimelineTabs } from '../TimelineTabs'

type Props = {
  test: string
}

class JungleMinds extends Component<Props> {
  render () {
    return (
      <div className='JungleMinds' data-test={this.props.test}>
        <Helmet
          title='Overview'
        />
        <Wrapper>
          <Navigation/>
          <MainContentGrid>
            <header>
              <PageTitle {...jungleMinds}/>
              <Intro {...jungleMinds}/>
              <TimelineTabs {...overallData} />
            </header>
          </MainContentGrid>
        </Wrapper>
      </div>
    )
  };
}

export default JungleMinds

const Wrapper = styled.div`
  display: flex;
`
