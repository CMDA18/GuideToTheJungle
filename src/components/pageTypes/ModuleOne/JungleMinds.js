// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import TextAndImage from '../../TextAndImage'
import { jungleMinds } from '../../../constants/content/index'

type Props = {
  test: string
}

class JungleMinds extends Component<Props> {
  render () {
    return (
      <div className='JungleMinds' data-test={this.props.test}>
        <Helmet
          title='JungleMinds'
        />
        <Grid page='jungleMinds'>
          <header>
            <PageTitle {...jungleMinds}/>
            <Intro {...jungleMinds}/>
          </header>
          <TextAndImage {...jungleMinds.textMedia} />

        </Grid>
      </div>
    )
  }
}

export default JungleMinds
