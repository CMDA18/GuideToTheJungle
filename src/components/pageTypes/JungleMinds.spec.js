import React from 'react'
import { shallow } from 'enzyme'
import JungleMinds from './JungleMinds'

it('renders without crashing', () => {
  expect(shallow(<JungleMinds />)).toMatchSnapshot()
})
