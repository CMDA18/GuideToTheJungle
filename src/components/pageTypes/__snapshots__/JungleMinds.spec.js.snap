// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <JungleMinds />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="Overview"
        />,
        <JungleMinds__Wrapper>
          <Navigation
            page="jungleMinds"
          />
          <MainContentGrid>
            <header>
              <PageTitle
                introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                introTitle="Our evolution in time"
                pageTitle="0.1 Jungle minds"
              />
              <Intro
                introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                introTitle="Our evolution in time"
                pageTitle="0.1 Jungle minds"
              />
              <TimelineTabs
                0={
                  Object {
                    "content": Object {
                      "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                      "title": "Our evolution in time",
                    },
                    "name": "Today",
                  }
                }
                1={
                  Object {
                    "content": Object {
                      "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                      "title": "Our evolution in time",
                    },
                    "name": "Tomorrow",
                  }
                }
                2={
                  Object {
                    "content": Object {
                      "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                      "title": "Our evolution in time",
                    },
                    "name": "Beyond",
                  }
                }
              />
            </header>
          </MainContentGrid>
        </JungleMinds__Wrapper>,
      ],
      "className": "JungleMinds",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "Overview",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Navigation
              page="jungleMinds"
            />,
            <MainContentGrid>
              <header>
                <PageTitle
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <Intro
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <TimelineTabs
                  0={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Today",
                    }
                  }
                  1={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Tomorrow",
                    }
                  }
                  2={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Beyond",
                    }
                  }
                />
              </header>
            </MainContentGrid>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "page": "jungleMinds",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <header>
                <PageTitle
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <Intro
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <TimelineTabs
                  0={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Today",
                    }
                  }
                  1={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Tomorrow",
                    }
                  }
                  2={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Beyond",
                    }
                  }
                />
              </header>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <PageTitle
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />,
                  <Intro
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />,
                  <TimelineTabs
                    0={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Today",
                      }
                    }
                    1={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Tomorrow",
                      }
                    }
                    2={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Beyond",
                      }
                    }
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "introText": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                    "introTitle": "Our evolution in time",
                    "pageTitle": "0.1 Jungle minds",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "introText": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                    "introTitle": "Our evolution in time",
                    "pageTitle": "0.1 Jungle minds",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "0": Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Today",
                    },
                    "1": Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Tomorrow",
                    },
                    "2": Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Beyond",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "header",
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="Overview"
          />,
          <JungleMinds__Wrapper>
            <Navigation
              page="jungleMinds"
            />
            <MainContentGrid>
              <header>
                <PageTitle
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <Intro
                  introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                  introTitle="Our evolution in time"
                  pageTitle="0.1 Jungle minds"
                />
                <TimelineTabs
                  0={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Today",
                    }
                  }
                  1={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Tomorrow",
                    }
                  }
                  2={
                    Object {
                      "content": Object {
                        "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                        "title": "Our evolution in time",
                      },
                      "name": "Beyond",
                    }
                  }
                />
              </header>
            </MainContentGrid>
          </JungleMinds__Wrapper>,
        ],
        "className": "JungleMinds",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "Overview",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Navigation
                page="jungleMinds"
              />,
              <MainContentGrid>
                <header>
                  <PageTitle
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />
                  <Intro
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />
                  <TimelineTabs
                    0={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Today",
                      }
                    }
                    1={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Tomorrow",
                      }
                    }
                    2={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Beyond",
                      }
                    }
                  />
                </header>
              </MainContentGrid>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "page": "jungleMinds",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <header>
                  <PageTitle
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />
                  <Intro
                    introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                    introTitle="Our evolution in time"
                    pageTitle="0.1 Jungle minds"
                  />
                  <TimelineTabs
                    0={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Today",
                      }
                    }
                    1={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Tomorrow",
                      }
                    }
                    2={
                      Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Beyond",
                      }
                    }
                  />
                </header>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <PageTitle
                      introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                      introTitle="Our evolution in time"
                      pageTitle="0.1 Jungle minds"
                    />,
                    <Intro
                      introText="Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life."
                      introTitle="Our evolution in time"
                      pageTitle="0.1 Jungle minds"
                    />,
                    <TimelineTabs
                      0={
                        Object {
                          "content": Object {
                            "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                            "title": "Our evolution in time",
                          },
                          "name": "Today",
                        }
                      }
                      1={
                        Object {
                          "content": Object {
                            "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                            "title": "Our evolution in time",
                          },
                          "name": "Tomorrow",
                        }
                      }
                      2={
                        Object {
                          "content": Object {
                            "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                            "title": "Our evolution in time",
                          },
                          "name": "Beyond",
                        }
                      }
                    />,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "introText": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                      "introTitle": "Our evolution in time",
                      "pageTitle": "0.1 Jungle minds",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "introText": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                      "introTitle": "Our evolution in time",
                      "pageTitle": "0.1 Jungle minds",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "0": Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Today",
                      },
                      "1": Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Tomorrow",
                      },
                      "2": Object {
                        "content": Object {
                          "text": "Our company is founded in 2000 as the first user centric company. Since then we keep deserving this title by making products that really matters in a person's life.",
                          "title": "Our evolution in time",
                        },
                        "name": "Beyond",
                      },
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "header",
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
